src/utils/resources/vm/utils/network/constants.ts

```diff
@@ -12,10 +12,10 @@ const typeHandler = {
   },
 };
 
-const types = {
+export const NetworkTypes = {
```

Comment:
no need to export it. u can simply use import interfaceTypes, it is a proxy, you will get a value from type if exisit if not u will get Bridge as the default value.